src/index.ts

```diff
@@ -57,7 +57,7 @@ export class Duffel {
     this.aircraft = new Aircraft(this.client)
     this.airlines = new Airlines(this.client)
     this.airports = new Airports(this.client)
-    this.links = new Links(this.client)
+    this.links = new Sessions(this.client)
```

Comment:
We've decided to expose this under the `links` key to make it obvious. This is not consistent with the rest of the API. We should at some point think about making this consistent.